pcf/test/particle/aws/testdata.json

```diff
@@ -471,10 +471,36 @@
       "mock_s3"
     ]
   ],
+  "sns-apply": [
```

Comment:
if you are using the automated moto tests as part of the testing framework then you dont need to add any seperate test files.